### PR TITLE
24 activity get dictionary value

### PR DESCRIPTION
**Pull request by dc500-code**

### CODE REVIEW

File: README.md
Comment (k2zinger) [inline README.md:1]: move location (alphabetical order)

File: Shared/Themes/Generic.xaml
Comment (k2zinger) [inline Shared/Themes/Generic.xaml:1]: revert changes to shared files

File: Shared/UiPath.Shared.Activities.Design/Controls/FilePathControl.xaml
Comment (k2zinger) [inline Shared/UiPath.Shared.Activities.Design/Controls/FilePathControl.xaml:1]: revert changes to shared files

File: Shared/UiPath.Shared.Activities.Design/Controls/RestrictiveBodyConnector.xaml
Comment (k2zinger) [inline Shared/UiPath.Shared.Activities.Design/Controls/RestrictiveBodyConnector.xaml:1]: revert changes to shared files

File: Shared/UiPath.Shared.Activities.Design/Editors/EditorTemplates.xaml
Comment (k2zinger) [inline Shared/UiPath.Shared.Activities.Design/Editors/EditorTemplates.xaml:1]: revert changes to shared files

File: UiPathTeam.Extensions.Tests/DictionaryUnitTests.cs [hunk 1]
```diff
@@ -81,7 +81,7 @@ public void TestRemoveFromDictionary()
         public void TestDictionaryContainsValue()
         {
             var dictionary = new Dictionary<Object, Object>();
-            var boolResult = true;
+            var boolResult = false;
```
Comment (k2zinger) [inline UiPathTeam.Extensions.Tests/DictionaryUnitTests.cs:84]: why was this changed?  this test is unrelated to what you wrote

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities.Design/Designers/AddToDictionaryDesigner.xaml
Comment (k2zinger) [inline UiPathTeam.Extensions/UiPathTeam.Extensions.Activities.Design/Designers/AddToDictionaryDesigner.xaml:1]: revert changes to unrelated files

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities.Design/Designers/DictionaryContainsValueDesigner.xaml
Comment (k2zinger) [inline UiPathTeam.Extensions/UiPathTeam.Extensions.Activities.Design/Designers/DictionaryContainsValueDesigner.xaml:1]: revert changes to unrelated files

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities.Design/Themes/Icons.xaml
Comment (k2zinger) [inline UiPathTeam.Extensions/UiPathTeam.Extensions.Activities.Design/Themes/Icons.xaml:1]: revert changes to unrelated files

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Activities/GetDictionaryValue.cs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        }
+
+        protected override async Task<Action<AsyncCodeActivityContext>> ExecuteAsync(AsyncCodeActivityContext context, CancellationToken cancellationToken)
+        {
+            // Inputs
+            var dictionary = Dictionary.Get(context);
+            var key = Key.Get(context);
+            var boolresult = false;
+
+            var result = Result.Get(context);
```
Comment (k2zinger) [inline UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Activities/GetDictionaryValue.cs:70]: do not assign a null value to a var, define the variable instead
object result;

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx
Comment (k2zinger) [inline UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx:1]: remove duplicates
e.g. Common_Category, Input_Category, etc.

**Review by k2zinger — CHANGES_REQUESTED** [review]

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Activities/GetDictionaryValue.cs (hunk 2 above)
Comment (dc500-code) [reply UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Activities/GetDictionaryValue.cs]: done

File: README.md
Comment (dc500-code) [reply README.md:1]: Done

**Review by k2zinger — APPROVED** [review]
